### PR TITLE
From home

### PR DESCRIPTION
**Pull request by ruvdrsti**
In the latex file I wrote down to what degree I currently understand the given paper. If you could give me some pointers as to where I still need some improvements, that would be greatly appreciated.

### CODE REVIEW

File: checkpoint_CUHF.tex [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+\begin{itemize}
+    \item no unique Fock operator
+    \item hard to interpret uniformly
+\end{itemize}
+ When looking at the Fock operator, we see equations like Equation \eqref{eq: ROHF fock}.
+ \begin{equation}\label{eq: ROHF fock}
+     R_{cc} = A_{cc}F^\alpha_{cc} + B_{cc}F^\beta_{cc}
+ \end{equation}
+ Since fixed values for the A and B factors exist, the Fock operator and consequently its eigenvalues can vary depending on these values. However, the wave function and total energy 
+ remain the same. This make the interpretation of the eigenvalues difficult. They no longer follow Koopmans' theorem.
```
Comment (johdvos) [inline checkpoint_CUHF.tex:26]: If you would like additional reading on Koopmans' theorem in ROHF, this [publication](https://aip.scitation.org/doi/abs/10.1063/1.2393223) deals with this topic.

File: checkpoint_CUHF.tex [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+\subsection{Unrestricted Hartee-Fock}
```
Comment (johdvos) [inline checkpoint_CUHF.tex:27]: Make sure to understand what you are giving up in exchange for what.

**Review by johdvos — APPROVED** [review]
Your notes seem fine. I left two comments you could have a look at. For the upcoming meeting about this I suggest the following:

- Try to work out the paper from start to finish, with the intermediate results you have given. This will help you really understand what's going on.
- Make sure you understand what the implications are of using this NO basis.
- Try to work out a general scheme of the steps an implementation should need. A general, qualitative idea is more than sufficient.

File: checkpoint_CUHF.tex [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+\label{subsec:UHF}
+\begin{itemize}
+    \item spin-contamination
+    \item loss of information
+\end{itemize}
+When orbitals are allowed to vary, we will see overlap terms in the expectation value of the $S^2$ operator. This results in the loss of a quantum number, since the wave function
```
Comment (AaronDC60) [inline checkpoint_CUHF.tex:33]: Why is spin contamination and thus losing this quantum number so bad?

**Review by AaronDC60 — COMMENTED** [review]
Great work so far.

**Comment by ruvdrsti** [comment]
Sorry for the delay, I have been busy tying up some other loose ends as well, but I have addressed your questions. The items for the meeting have been done on paper (except for the implementation) and I have not yet added to this PR, but I could still do so if you wish.
Thanks for the feedback by the way.

**Review by johdvos — APPROVED** [review]

**Review by MFatjano — APPROVED** [review]

**Pull request merged**